### PR TITLE
[Frequency analyzer part 4] Save frequency distribution plot

### PR DESCRIPTION
**Pull request by huangxichen1**

### CODE REVIEW

**Review by jiayu-google — APPROVED** [review]
Thanks Xichen!

**Review by my2457 — APPROVED** [review]
lgtm

**Review by shengma-1 — APPROVED** [review]

File: src/evaluations/tests/analyzer_test.py [hunk 1]
```diff
@@ -258,6 +258,59 @@ def test_get_analysis_results(self):
...
+                        num_sets=3,
+                        set_size=50,
+                        freq_rates=[5] * 3,
+                        freq_cap=8,
+                    )))]
+    )
+
+    self.run_name = 'test_run'
+
+    def _get_test_evaluator(out_dir):
```
Comment (kungfucraig) [inline src/evaluations/tests/analyzer_test.py:295]: Don't intersperse helper methods in code. Either make module level or put at the top of the function.

**Review by kungfucraig — APPROVED** [review]

File: src/evaluations/tests/analyzer_test.py (hunk 1 above)
Comment (huangxichen1) [reply src/evaluations/tests/analyzer_test.py]: Done.

**Pull request merged**